zookeeper-jute/src/main/java/org/apache/jute/compiler/JRecord.java

```diff
@@ -767,4 +780,30 @@ public static String getCsharpFQName(String name) {
         }
         return fQName.toString();
     }
+
+    protected String getRecordComments() {
+        if (mRecordToken == null || mRecordToken.specialToken == null) {
+            return null;
+        }
+
+        StringJoiner joiner = new StringJoiner("\n * ", "/**\n * ", "\n */\n");
+        Token tmpTkn = mRecordToken.specialToken;
+        while (tmpTkn.specialToken != null) {
+            tmpTkn = tmpTkn.specialToken;
+        }
+
+        while (tmpTkn != null) {
+            String image = tmpTkn.image.trim();
+            tmpTkn = tmpTkn.next;
+            if (image.startsWith("//")) {
```

Comment:
The capture does not shaped well for directly usage. I tasted the following capture, and it captured a complete line comment. This way we can output whatever it captured without post work.

```
SPECIAL_TOKEN :
{
  <SINGLE_LINE_COMMENT: "//" (~["\n","\r"])* ("\n"|"\r"|"\r\n")>
}
```

See also https://javacc.github.io/javacc/tutorials/token-manager.html

Reply:
Thank you very much for your comment. I will make some further modifications.😆

Reply:
> The capture does not shaped well for directly usage. I tasted the following capture, and it captured a complete line comment. This way we can output whatever it captured without post work.
> 
> ```
> SPECIAL_TOKEN :
> {
>   <SINGLE_LINE_COMMENT: "//" (~["\n","\r"])* ("\n"|"\r"|"\r\n")>
> }
> ```
> 
> See also https://javacc.github.io/javacc/tutorials/token-manager.html

It seems like there's no need to change this now, because I get all the comments，then I use and distinguished them by line numbe. 😂😂